workflow/common/common.go

```diff
@@ -139,7 +139,7 @@ const (
 	// GlobalVarWorkflowDuration is the current duration of this workflow
 	GlobalVarWorkflowDuration = "workflow.duration"
 	// GlobalVarWorkflowParameters is a JSON string containing all workflow parameters
-	GlobalVarWorkflowParameters = "workflow.parameters"
+	GlobalVarWorkflowParameters = "workflow.parameters.json"
```

Comment:
Randomly `bellows.Expand` failed to expend  all global variable `workflow.parameters.xxx` if `workflow.parameters` expended first.  We should have a different name for `workflow.parameters` json format.

Reply:
we should raise a bug with bellows?

Reply:
maybe not, I think it does not make sense - but can we make sure that this is not used - new in v3.1?

Reply:
also - should work for fasttemplate

Reply:
`fasttemplate` will just string replace. it will work. I can ask the community channel for this variable.

Reply:
if it is in v3.0, then changing it would be breaking change

Reply:
I think we should exclude this from `{{=` templates. Users should not need it.

Reply:
we can prevent to have {{= in `when` element. So this is the way when will have only `govaluate` format.